### PR TITLE
fix: Add test case for topelevel tuple hover

### PR DESCRIPTION
**Pull request by tgodzik**
Fixes https://github.com/scalameta/metals/issues/2561

### CODE REVIEW

File: tests/cross/src/test/scala/tests/hover/HoverTermSuite.scala [hunk 1]
```diff
@@ -455,4 +455,20 @@ class HoverTermSuite extends BaseHoverSuite {
...
+    "toplevel".tag(IgnoreScala2),
+    """
+      |val (first, <<se@@cond>>) = (1, false)
+      |""".stripMargin,
+    """|```scala
+       |package 
+       |```
+       |""".stripMargin,
+    compat = Map(
+      "3.1.3" ->
```
Comment (tanishiking) [inline tests/cross/src/test/scala/tests/hover/HoverTermSuite.scala:469]: Maybe we should be able to express "3.1.3 or higher" by `compat` (of course we don't need to handle this PR)?
Looks like now we have to add `3.1.4`, `3.1.5`, ... patterns for every new scala3 release.

**Review by tanishiking — APPROVED** [review]
Oh, it's already fixed in https://github.com/lampepfl/dotty/issues/11919 great job :tada:
LGTM

File: tests/cross/src/test/scala/tests/hover/HoverTermSuite.scala (hunk 1 above)
Comment (kpodsiad) [reply tests/cross/src/test/scala/tests/hover/HoverTermSuite.scala:469]: Maybe we can use https://github.com/vdurmont/semver4j

File: tests/cross/src/test/scala/tests/hover/HoverTermSuite.scala [hunk 2]
```diff
@@ -455,4 +455,20 @@ class HoverTermSuite extends BaseHoverSuite {
...
+
+  check(
+    "toplevel".tag(IgnoreScala2),
+    """
+      |val (first, <<se@@cond>>) = (1, false)
+      |""".stripMargin,
+    """|```scala
+       |package 
+       |```
+       |""".stripMargin,
```
Comment (kpodsiad) [inline tests/cross/src/test/scala/tests/hover/HoverTermSuite.scala:467]: Could you align pipes? That offset really bothers me 😆

**Review by kpodsiad — APPROVED** [review]

File: tests/cross/src/test/scala/tests/hover/HoverTermSuite.scala (hunk 2 above)
Comment (tgodzik) [reply tests/cross/src/test/scala/tests/hover/HoverTermSuite.scala]: Fixed!

File: tests/cross/src/test/scala/tests/hover/HoverTermSuite.scala (hunk 1 above)
Comment (tgodzik) [reply tests/cross/src/test/scala/tests/hover/HoverTermSuite.scala]: We already have some mechanisms for doing this as well as out own `SemVer`, so I just reused that.

**Pull request merged**